docker/jetstream-pytorch-server/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Ubuntu:22.04
+# Use Ubuntu 22.04 from Docker Hub.
+# https://hub.docker.com/_/ubuntu/tags?page=1&name=22.04
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PYTORCH_JETSTREAM_VERSION=main
+
+RUN apt -y update && apt install -y --no-install-recommends \
+    ca-certificates \
+    git \
+    python3.10 \
+    python3-pip
+
+RUN python3 -m pip install --upgrade pip
+
+RUN update-alternatives --install \
+    /usr/bin/python3 python3 /usr/bin/python3.10 1
+
+
+RUN git clone https://github.com/AI-Hypercomputer/JetStream.git
```

Comment:
remove this line. Jetstream is installed as part of install_everything.sh

Reply:
JetStream is part of `install_everything.sh` but its reference is an older version. For our use case, we just want to build off of main for both JetStream and jetstream-pytorch.

Reply:
i see, if that is the desire, then, we need to update the version that `jetstream-pytorch` pins. Because the CIs is running this version otherwise it is not guaranteed to work ( especially when the proto definition changes.

Reply:
You can do that in the same PR. do `cd deps/JetStream` then `git pull` then `git checkout <hash or tag or branch>`. Then go back to the root and commit the submodule.

Reply:
Ack, updated the JetStream dependency